### ISSUE TITLE
Not being able to pass paging params for ticket searches?

### ISSUE THREAD

**Issue by chemsafe** [dbasoftware/freshdesksharp#6]
Hi. As far as I see, there is no way to pass a paging param for ticket searches. Am I missing something?

I ended up adding in FreshDeskTicketsEndpoint.cs:

```
        public async Task<FreshdeskTicketSearchResults<FreshdeskCustomFields>> SearchAsync(string searchQuery,string page)
        {
            return await SearchAsync<FreshdeskCustomFields>(searchQuery, page).ConfigureAwait(false);
        }

        public async Task<FreshdeskTicketSearchResults<TCustomFieldObject>> SearchAsync<TCustomFieldObject>(string searchQuery,string page) where TCustomFieldObject : class
        {
            var query = $"?query=\"{WebUtility.UrlEncode(searchQuery)}\"";
            var requestUri = $"{_apiBaseUri}/search/tickets{query}?page={page}";
            using (var response = await _httpClient.GetAsync(requestUri).ConfigureAwait(false))
            {
                return await GetResponseAsync<FreshdeskTicketSearchResults<TCustomFieldObject>>(response).ConfigureAwait(false);
            }
        }

```

**Comment by fyndor** [comment]
I don't have a way to test it as I no longer use Freshdesk or have an active account, but I have added your changes.  If you notice your suggested change, it starts the query string (with ?) twice.  I implemented it slightly differently.  I made the page an optional int parameter so it doesn't break people updating and I also added the per_page option which is part of the current API.  If you see this and get a chance, grab my latest changes and see if it works as intended.  I get nervous updating this library as I don't want to break anyone using it, but I don't have a way to test my changes anymore.  It compiles obviously and looks right.  Hopefully I didn't make any mistakes I didn't notice.
